default.nix

```diff
@@ -35,7 +36,9 @@ let
   components = with pkgs; lib.makeScope newScope (self: with self; {
 
     inherit updateUrl deployUrl kioskUrl;
-    inherit (application) version safeProductName fullProductName;
+    inherit (application) safeProductName fullProductName;
+
+    version = application.version + (lib.optionalString buildTest "-TEST");
```

Comment:
Not against this, but is it strictly needed, or a helper in identifying some output, or ...?

Reply:
Just a precaution to avoid somehow somewhere using/deploying the version as if it was a real build. I would keep it, makes it clear this is a special build.